packages/kafka-iam-auth/src/index.ts

```diff
@@ -1,14 +1,17 @@
-/* eslint-disable no-constant-condition */
-export * from "./constants.js";
-export * from "./create-authenticator.js";
-export * from "./create-mechanism.js";
-export * from "./create-payload.js";
-export * from "./create-sasl-authentication-request.js";
-export * from "./create-sasl-authentication-response.js";
-import { Consumer, EachMessagePayload, Kafka } from "kafkajs";
-import { KafkaConsumerConfig, genericLogger } from "pagopa-interop-commons";
+import { generateAuthToken } from "aws-msk-iam-sasl-signer-js";
+import {
+  Consumer,
+  EachMessagePayload,
+  Kafka,
+  KafkaConfig,
+  OauthbearerProviderResponse,
+} from "kafkajs";
+import {
+  KafkaConsumerConfig,
+  Logger,
+  genericLogger,
+} from "pagopa-interop-commons";
 import { kafkaMessageProcessError } from "pagopa-interop-models";
-import { createMechanism } from "./create-mechanism.js";
 
 export const DEFAULT_AUTHENTICATION_TIMEOUT = 60 * 60 * 1000;
 export const REAUTHENTICATION_THRESHOLD = 20 * 1000;
```

Comment:
it seems a good candidate for an env variable.
May I suggest to move this to the config using 20 seconds as default value?
So we can tune it in the environment only in case of need